ddl/ddl.go

```diff
@@ -188,11 +198,18 @@ type DDL interface {
 	GetInfoSchemaWithInterceptor(ctx sessionctx.Context) infoschema.InfoSchema
 	// DoDDLJob does the DDL job, it's exported for test.
 	DoDDLJob(ctx sessionctx.Context, job *model.Job) error
+	// MigrateExistingDDLs move existing DDLs from queue to table.
+	MigrateExistingDDLs(bool) error
+	// BackOffDDLs move existing DDLs from table to queue.
+	BackOffDDLs() error
```

Comment:
The naming of these two methods is not good. How about `MoveJobToTable` and `MoveJobToQueue`?

BTW, I like putting everything to the interface when I was young. Then I was accused... https://www.zenlife.tk/stop-add-method-to-interface.md

Reply:
we can only get an interface at the used place